src/frontend/frontend.py

```diff
@@ -643,7 +643,8 @@ def format_currency(int_amount):
         os.environ.get('CONTACTS_API_ADDR'))
     app.config['PUBLIC_KEY'] = open(os.environ.get('PUB_KEY_PATH'), 'r').read()
     app.config['LOCAL_ROUTING'] = os.getenv('LOCAL_ROUTING_NUM')
-    app.config['BACKEND_TIMEOUT'] = 4  # timeout in seconds for calls to the backend
+    # timeout in seconds for calls to the backend
+    app.config['BACKEND_TIMEOUT'] = int(os.getenv('BACKEND_TIMEOUT', '5'))
```

Comment:
I have faced large numbers of errors when `loadgenerator` call signup requests due to timeout to `userservice` in `frontend`. This is because `userservice` requires more time to process requests. My idea was to increase `BACKEND_TIMEOUT` to reduce the number of errors and to avoid building a docker image each time when I need to change this setting, so I decide to define env var for this. But unfortunately increasing this timeout will lead to bigger latency. So I left this change just a small improvement.

Reply:
> increasing this timeout will lead to bigger latency

So increasing this timeout from 4 to 5 leads to bigger latencies? In that case can we stay at 4, but have it be an env like you suggested to easily be able to modify it?

Reply:
The bigger the timeout the bigger the response latency to the user. Yes, I will change back default value to 4.